labkey-client-api/build.gradle

```diff
@@ -225,37 +213,6 @@ project.publishing {
             }
         }
     }
-    apply plugin: 'com.jfrog.artifactory'
-    artifactory {
-        contextUrl = "${artifactory_contextUrl}"
-        //The base Artifactory URL if not overridden by the publisher/resolver
-        publish {
-            repository {
-                repoKey = BuildUtils.getRepositoryKey(project)
-                if (project.hasProperty('artifactory_user') && project.hasProperty('artifactory_password'))
-                {
-                    username = artifactory_user
-                    password = artifactory_password
-                }
-                maven = true
-            }
-            defaults {
-                publishBuildInfo = false
-                publishPom = true
-                publishIvy = false
-            }
-        }
-    }
-
-    project.artifactoryPublish {
```

Comment:
TeamCity seems unhappy about this removal

Reply:
Yes, I need to make adjustments there, as noted in my descriptions.

Reply:
Right, sorry, missed that